### PR TITLE
Add missing anchors and move some from h2 to section tag

### PR DESCRIPTION
**Pull request by zimeon**
Noticed missing section links in the versioning section in particular...

### CODE REVIEW

**Review by barmintor — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -222,13 +222,13 @@
...
       </dl>
-  </section>
+    </section>
     <section id="resource-management">
       <h2>Resource Management</h2>
 
       <section id="general">
         <h2>General</h2>
-	<section id="ldpnr-ixn-model">
+	        <section id="ldpnr-ixn-model">
```
Comment (dannylamb) [inline index.html]: Indentation looks off here.

Comment (zimeon) [reply index.html]: Rats, was fixing bad indent and managed to put a tab in...

**Review by awoods — APPROVED** [review]
Thanks for this clean-up. There is probably some minor value in having a consistent approach to anchor syntax, such as "all lowercase with dashes between words"...
...but this PR is an improvement.

**Review by dannylamb — DISMISSED** [review]

**Comment by zimeon** [comment]
@awoods - I agree that a consistent policy would probably be good but with this PR I just want to move the baseline up a bit without wholesale changing the existing anchors

**Review by awoods — DISMISSED** [review]

**Pull request merged**